addons/radar/functions/fnc_getIcon.sqf

```diff
@@ -63,8 +63,14 @@ if (getText(configFile >> "CfgWeapons" >> (secondaryWeapon (_unit)) >> "UIPictur
 };
 
 // Medic
-if (_unit getVariable ["ace_medical_medicClass", [0, 1] select (_unit getUnitTrait "medic")] > 0) exitWith {
-    _namespace getVariable ["medic", DUI_MEDIC];
+if (diwako_dui_toggleAceMedic) then {
```

Comment:
I kind of disagree with this. The current implementation uses both, ACE set medics AND vanilla set medics. Having a whitelist for recognized medic classes introduces some issues.

Each side have their own medic classes, on top of that each faction have their own medic classes. The check that has been added looks for EXACTLY that unit class and not if it inherits from one of the whitelisted ones. It also checks for case sensitivity which many many people get wrong very often.

Besides. The only added default unit class will get the vanilla medic trait applied, thus it would be redundant.

The ACE check should only be visible in the CBA settings if ACE Medical is loaded and available. 
Then it should only alter how a medic is being recognized. As in, it should only check the vanilla trait and not the ACE variable on the unit.
Then it is way easier for mission makers to make a none medic class a medic, by just giving them the vanilla trait, People that want all units to be ACE medics can just set them and only the ones with actual medic traits get the icon, while everyone can do everything.